rosetta-source/src/main/rosetta/base-staticdata-asset-common-type.rosetta

```diff
@@ -5,17 +5,39 @@ import cdm.base.*
 import cdm.base.datetime.*
 import cdm.base.math.*
 import cdm.base.staticdata.party.*
+import cdm.base.staticdata.identifier.*
 import cdm.observable.asset.*
 import cdm.product.template.*
 
+choice Asset:  <"An Asset is defined something that can be owned and transferred in the financial markets. As a choice data type, one and only one of the  attributes must be used. Cash: An Asset that consists solely of a monetary holding in a currency. Commodity: An Asset comprised of raw or refined materials or agricultural products, eg gold, oil or wheat. DigitalAsset An Asset that exists only in digital form, eg Bitcoin or Ethereum; excludes the digital representation of other Assets. Instrument: An asset that is issued by one party to one or more others; Instrument is also a choice data type.">
```

Comment:
An Asset is defined **as** something...

Reply:
also does choice support inline comments, so we could put the comment for Cash next to the Cash item?

Reply:
"as" inserted.
`choice` didn't support inline comments, but now it does.  Updated.